src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        if (driverDao.get(id).isEmpty()) {
+            throw new NoSuchElementException("Can't get driver with id " + id);
+        }
+        return driverDao.get(id).get();
```

Comment:
What about this option?)
```suggestion
        return driverDao.orElseThrow(() -> new NoSuchElementException("Can't get driver with id " + id));
```